.github/workflows/ci.yml

```
@@ -1,15 +1,36 @@
     branches: [main]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   build:
     runs-on: ubuntu-latest
+    name: Execute project build
+    steps:
+      - name: Checkout
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)